packages/Liquid-Tests.package/LQPollTests.class/instance/testClipboardImport.st

```diff
@@ -1,5 +1,6 @@
 tests
 testClipboardImport
+	"The functionality tested here is no longer used by Liquid. Leaving the test since it should still work"
 	| poll randomID client |
 	client := LocalObjectRepo new.
 	randomID := UUID new asString.
```

Comment:
Sand: Why are we leaving in the morning this test? Does it have any value except for increasing the number of tests by one? I'd remove it

Reply:
I wasn't sure myself if we should leave it or not. As the comment says we don't depend on the test working, but my thought behind leaving it in was that it doesn't hurt either - who knows, a future team might implement some other version of this. But if you think it would look better if we removed the test, I have no problem with that either

Reply:
IMO it's best to remove code that doesn't add any value to the project. I looked at this test more closely and I think it's just testIfPresentIfAbsent but using the clipboard as a variable I don't really see how that would help a future team. Btw I typed the last comment from the app, nice example of broken autocomplete^^